README.md

```diff
@@ -51,15 +55,14 @@ The Auto Advance feature has the following configuration options:
 
 ## Search and Recommendations
 
-If you're a customer who has [Data-Driven recommendations](https://support.jwplayer.com/customer/en/portal/articles/2191721-data-driven-recommendations-beta) enabled. You can make use of Search and Recommendations.
+If you're a customer who has [JW Recommendations](https://support.jwplayer.com/customer/portal/articles/2191721-jw-recommendations) enabled (formerly called Feeds), you can make use of Search and Recommended playlists. If you have a Search playlist specified, a Search box will appear in your Apple TV app UI. The Recommended playlist will be used to suggest related videos when a user selects a search result.
 
-In order to do this, you need to set up a Search Feed and a Recommendations Feed in your `config.json` file.
-At this moment it's possible to create Recommendation feeds through your dashboard, but you'll have to create a search feed through the [API](https://developer.jwplayer.com/jw-platform/reference/v1/methods/channels/create.html). Please contact us if you need help setting this up.
+To enable search, you must specify a Search playlist ID and a Related playlist ID in your `config.json` file. Make sure you have set up Search and Recommended playlists through your JW Dashboard or through the [Platform API](https://developer.jwplayer.com/jw-platform/reference/v1/methods/channels/create.html).
```

Comment:
Might want to reference the actual property names of the keys we are going to use here.